src/Core/Grid/Definition/Factory/LogGridDefinitionFactory.php

```diff
@@ -265,4 +268,14 @@ protected function getGridActions()
                     ->setIcon('storage')
             );
     }
+
+    private function getSeveritysChoices()
+    {
+        return [
+            $this->trans('Informative only', [], 'Admin.Advparameters.Help') => 1,
+            $this->trans('Warning', [], 'Admin.Advparameters.Help') => 2,
+            $this->trans('Error', [], 'Admin.Advparameters.Help') => 3,
+            $this->trans('Major issue (crash)!', [], 'Admin.Advparameters.Help') => 4,
```

Comment:
Can be useful to set constants for Severity Levels somewhere

Reply:
Very good ideia ! Thanks @Matt75